.circleci/config.yml

```diff
@@ -11,10 +11,11 @@ jobs:
       - save_cache:
           key: dependency-cache-{{ checksum "pom.xml" }}
           paths: ~/.m2
+      - run: mvn -X scoverage:report
       - run:
           name: sonar
           command: |
-                 mvn verify -Dlog4j.configuration=./logs sonar:sonar -Dsonar.projectKey=project-sunbird_sunbird-data-pipeline -Dsonar.organization=project-sunbird -Dsonar.exclusions=ep-core/src/main/java/org/ekstep/ep/samza/core/Logger.java -Dsonar.host.url=https://sonarcloud.io -Dsonar.coverage.jacoco.xmlReportPaths=/home/circleci/dp/data-pipeline/assessment-aggregator/target/coverage-reports/jacoco-ut/jacoco.xml,/home/circleci/dp/data-pipeline/content-cache-updater/target/coverage-reports/jacoco-ut/jacoco.xml,/home/circleci/dp/data-pipeline/de-normalization/target/coverage-reports/jacoco-ut/jacoco.xml,/home/circleci/dp/data-pipeline/de-duplication/target/coverage-reports/jacoco-ut/jacoco.xml,/home/circleci/dp/data-pipeline/derived-de-duplication/target/coverage-reports/jacoco-ut/jacoco.xml,/home/circleci/dp/data-pipeline/device-profile-updater/target/coverage-reports/jacoco-ut/jacoco.xml,/home/circleci/dp/data-pipeline/druid-events-validator/target/coverage-reports/jacoco-ut/jacoco.xml,/home/circleci/dp/data-pipeline/ep-core/target/coverage-reports/jacoco-ut/jacoco.xml,/home/circleci/dp/data-pipeline/ep-telemetry-reader/target/coverage-reports/jacoco-ut/jacoco.xml,/home/circleci/dp/data-pipeline/events-router/target/coverage-reports/jacoco-ut/jacoco.xml,/home/circleci/dp/data-pipeline/telemetry-extractor/target/coverage-reports/jacoco-ut/jacoco.xml,/home/circleci/dp/data-pipeline/telemetry-location-updater/target/coverage-reports/jacoco-ut/jacoco.xml,/home/circleci/dp/data-pipeline/telemetry-router/target/coverage-reports/jacoco-ut/jacoco.xml,/home/circleci/dp/data-pipeline/telemetry-validator/target/coverage-reports/jacoco-ut/jacoco.xml,/home/circleci/dp/data-pipeline/user-cache-updater/target/coverage-reports/jacoco-ut/jacoco.xml,/home/circleci/dp/data-pipeline/share-events-flattener/target/coverage-reports/jacoco-ut/jacoco.xml
+                 mvn -X sonar:sonar -Dlog4j.configuration=./logs sonar:sonar -Dsonar.projectKey=project-sunbird_sunbird-data-pipeline -Dsonar.organization=project-sunbird -Dsonar.host.url=https://sonarcloud.io -Dsonar.scala.coverage.reportPaths=/home/circleci/dp/data-pipeline-flink/telemetry-extractor/target/scoverage.xml,/home/circleci/dp/data-pipeline-flink/dp-core/target/scoverage.xml
```

Comment:
The job would generate one file and use that. Check the example from sunbird-analytics-core or service repositories